modal/create_bingx_account_modal.py

```diff
@@ -0,0 +1,24 @@
+import discord
+import message as ms
+from discord.interactions import Interaction
+
+TITLE = "Submit Your Bing X Account Details"
+
+
+class CreateBingXAccountModal(discord.ui.Modal, title=TITLE):
+
+    def __init__(self, dbcon, selected_account):
+        self.dbcon = dbcon
+        self.user_account_name = selected_account
+        super().__init__(title=TITLE, timeout=120)
+
+    uuid = discord.ui.TextInput(label="Bing X UUID", placeholder="Bing X UUID", style=discord.TextStyle.short)
+
+    async def on_submit(self, interaction: Interaction):
+        uuid = self.uuid.value
+        discord_id = interaction.user.id
+        if self.dbcon.check_discord_id_exist_from_trade_volume_table(discord_id):
+            await interaction.response.send_message(ms.REGISTERED_DISCORD_ID, ephemeral=True)
```

Comment:
Can we change this to:
If account existed, update
Else insert

Reply:
fixed